tests/unit/BaseApiAbstractTest.php

```diff
@@ -28,7 +28,7 @@ public function testNoUserAgentExtensions()
         $this->assertNotFalse(
             \strpos(
                 $http_client->getConfig()['headers']['User-Agent'],
-                'smartling-api-sdk-php/3.9.2 (no extensions) GuzzleHttp/6'
+                'smartling-api-sdk-php/3.9.3 (no extensions) GuzzleHttp/6'
```

Comment:
Can you generate the final string using `CLIENT_LIB_ID_VERSION`? Just want to minimize such noisy changes.

Reply:
I believe it's fixed in `4.0.0-g7` branch. seems like `3.0.0-g6` is only used by Drupal for now. i don't expect more changes here in the nearest future